janitor/chemistry.py

```diff
@@ -114,17 +114,20 @@ def smiles2mol(
         import pandas as pd
         import janitor.chemistry
 
-        df = pd.DataFrame(...)
-
-        df = df.smiles2mol(smiles_column_name='smiles',
-                           mols_column_name='mols')
+        df = (
+            pd.DataFrame(...)
+            .smiles2mol(
+                smiles_column_name='smiles',
+                mols_column_name='mols'
+            )
+        )
 
     A progressbar can be optionally used.
 
     - Pass in "notebook" to show a tqdm notebook progressbar. (ipywidgets must
-      be enabled with your Jupyter installation.)
+    be enabled with your Jupyter installation.)
```

Comment:
It's a convention, mostly, nothing else. I haven't been strict about it, because there's no easy way to lint for it. But then again, I think there's a point to "if one can't lint for it, one shouldn't really bother"!